ts/utils/cookieManager.ts

```diff
@@ -1,14 +1,15 @@
 import CookieManager, { Cookie } from "@react-native-cookies/cookies";
 
-export const ioClearCookie = (cb: () => void) => {
-  CookieManager.clearAll().catch(cb);
-};
+export const ioClearCookie = (cb: () => void) =>
+  void CookieManager.clearAll().catch(cb);
+
+export const clearCookie = (url: string, name: string, cb?: () => void) =>
+  void CookieManager.clearByName(url, name).catch(_ => cb?.());
```

Comment:
`clearByName` is only available on iOS, we should remove cookies on Android too.

Reply:
Reading the library documentation, it seems that there is not a single API that is callable on both platforms in order to remove a previously set cookie. So, what do you say if, upon `clearCookie` invocation, we set the same cookie again, with both an invalid value (for the FIMS' token) and an hardcoded expired date (like `Fri, 1 Jan 2010 00:00:00 -0000`) ?

Reply:
Since it is not working properly, let us switch to `removeSessionCookies`. What do you think?

Reply:
Ok, agreed